hack/get_ocp_versions_for_testing.sh

```diff
@@ -4,7 +4,18 @@ set -euo pipefail
 
 __dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 PROJECT_ROOT_DIR="$(cd "$(dirname "${__dir}")" && pwd)"
-OPENSHIFT_SNO_IMAGE="quay.io/openshift-release-dev/ocp-release-nightly:4.8.0-0.nightly-2021-03-16-221720"
 
-# Append single node image as 4.8 for local deployments
-"${PROJECT_ROOT_DIR}"/tools/handle_ocp_versions.py --src ./default_ocp_versions.json --ocp-override ${OPENSHIFT_SNO_IMAGE} --name-override single-node-4.8-alpha --version-override 4.8 --skip-verify | tr -d "\n\t "
+# This is currently disabled because the 4.8-fc image has been released and we already
+# include it in `./default_ocp_versions.json`.
+# The code is still kept around behind this gate to provide an example of how we can do
+# a similar override in the future.
+OVERRIDE_4_8_WITH_SINGLE_NODE=0
+
+if [[ ${OVERRIDE_4_8_WITH_SINGLE_NODE} == 1 ]]; then
+    OPENSHIFT_SNO_IMAGE="quay.io/openshift-release-dev/ocp-release-nightly:4.8.0-0.nightly-2021-03-16-221720"
```

Comment:
why you are using the fc version here?

Reply:
this is now just meaningless old code used to serve as an example of how to do an override, it doesn't run, and that value doesn't matter